src/udev/udevadm-trigger.c

```diff
@@ -226,6 +267,8 @@ static int help(void) {
                "  -y --sysname-match=NAME           Trigger devices with this /sys path\n"
                "     --name-match=NAME              Trigger devices with this /dev name\n"
                "  -b --parent-match=NAME            Trigger devices with that parent device\n"
+               "  -e --entry-match                  Trigger devices having a udev db entry\n"
```

Comment:
hmm, "entry" is a bit too imprecise/generic. Maybe call this "dbentry"? or "database"? or even "initialized"? (i.e. the libudev/sd-device libs call the fact whether a device has a db entry "initialized"?

Reply:
I'm gonna change it to "--database-match" / "--database-nomatch". Maybe "initialized" would be even more consequent, since you mentioned libudev/sd-device libs call it this way, but I think this would sound equally generic to just calling it "entry".